azimuth/modules/dataset_analysis/dataset_warnings.py

```diff
@@ -80,6 +81,15 @@ def get_general_warnings(
         alert_eval_nb_min: List[bool] = eval_dist < min_num_per_class
         alert_nb_min: List[bool] = alert_train_nb_min + alert_eval_nb_min
 
+        train_mean = np.mean(train_dist)
+        eval_mean = np.mean(eval_dist)
+        perc_imb_train = (train_dist - train_mean) / train_mean
+        perc_imb_eval = (eval_dist - eval_mean) / eval_mean
+        max_delta_imb = self.config.dataset_warnings.max_delta_class_imbalance
+        alert_imb_train: List[bool] = np.abs(perc_imb_train) > max_delta_imb
+        alert_imb_eval: List[bool] = np.abs(perc_imb_eval) > max_delta_imb
+        alert_imb: List[bool] = alert_imb_train + alert_imb_eval
```

Comment:
Numpy seems to support a real OR operator, as documented [here](https://numpy.org/doc/stable/reference/generated/numpy.logical_or.html#:~:text=The%20%7C%20operator%20can%20be%20used%20as%20a%20shorthand%20for%20np.logical_or%20on%20boolean%20ndarrays.):
```suggestion
        alert_imb: List[bool] = alert_imb_train | alert_imb_eval
```

Reply:
By the way, are those 3 `List[bool]`s necessary? Isn't it able to infer the type automatically?

Reply:
No! They used to be, but `mypy` no longer complains. Good catch!!